### PR TITLE
Use cliquet wsgi middlewares

### PR DESCRIPTION
**Pull request by ametaireau**
Based on top of mozilla-services/cliquet#187

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
     'six>=1.9.0',
     'waitress>=0.8.9',
     'cliquet[postgresql,monitoring]>=1.6.0',
-    'raven'
```
Comment (ametaireau) [inline setup.py]: I'm actually not sure why raven is present here (shouldn't it be directly pulled off from cliquet?)

Comment (Natim) [inline setup.py]: Yes it is in the ``cliquet['monitoring']``

Comment (Natim) [inline setup.py]: We should remote it.

**Comment by ametaireau** [comment]
This needs cliquet 1.7 (not released yet)

**Pull request merged**